modules/Blockchain/Ethereum/index.js

```diff
@@ -52,6 +57,26 @@ class Ethereum {
         this.initalized = false;
     }
 
+    /**
+     * Reads identity from file
+     * @returns {Promise<erc725Identity>}
+     * @private
+     */
+    _loadIdentityFromFile() {
+        const implementation = this.config.blockchain.implementations
+            .filter(implementation => implementation.blockchain_id === blockchain_id);
+
+        const identityFilePath = path.join(
+            this.config.appDataPath,
+            implementation.identity_filepath,
+        );
+        if (fs.existsSync(identityFilePath)) {
+            const content = JSON.parse(fs.readFileSync(identityFilePath).toString());
+            return content.identity;
+        }
+        return null;
+    }
```

Comment:
Ethereum is an implementation, this kind of logic should be in the Blockchain.js module